README.md

```diff
@@ -1453,7 +1453,16 @@ export default adapter(
 );
 ```
 
-### Cloudflare (experimental)
+### Cloudflare Workers
+
+Waku projects can be deployed to Cloudflare Workers with [Wrangler](https://developers.cloudflare.com/workers/wrangler/).
+
+```sh
+CLOUDFLARE=1 npm run build
+wrangler deploy
+```
```

Comment:
Awesome.

Is it correct that we don't need `--template 07_cloudflare` for basic usage that doesn't use cloudflare specific features?

Reply:
Perfect.